tests/pyfunc/test_spark.py

```diff
@@ -103,7 +102,7 @@ def get_spark_session(conf):
     return (
         pyspark.sql.SparkSession.builder.config(conf=conf)
         .master(spark_master)
-        # .config("spark.driver.bindAddress", "127.0.0.1") # Uncomment for testing on M1 locally
+        # .config("spark.driver.bindAddress", "127.0.0.1")  # Uncomment for testing on M1 locally
```

Comment:
```suggestion
        # .config("spark.driver.bindAddress", "127.0.0.1")  # Uncomment for testing on Apple silicon locally
```

`Apple silicon` might be more accurate.

Reply:
agreed. changed